plugins/@grouparoo/clickhouse/src/lib/export/exportRecord.ts

```diff
@@ -0,0 +1,141 @@
+import { ExportRecordPluginMethod } from "@grouparoo/core";
+import { MySQLConnection } from "@grouparoo/mysql/dist/lib/connect";
+
+export const exportRecord: ExportRecordPluginMethod<MySQLConnection> = async ({
+  connection,
+  destination,
+  export: { newRecordProperties, oldRecordProperties, newGroups, toDelete },
+}) => {
+  let error;
+
+  let { table, primaryKey, groupsTable, groupForeignKey, groupColumnName } =
+    await destination.parameterizedOptions();
+
+  table = table?.toString();
+  primaryKey = primaryKey?.toString();
+  groupsTable = groupsTable?.toString();
+  groupForeignKey = groupForeignKey?.toString();
+  groupColumnName = groupColumnName?.toString();
+
+  if (Object.keys(newRecordProperties).length === 0) {
```

Comment:
```suggestion
  if (!Object.keys(newRecordProperties).length) {
```
Purely stylistic choice.